_pages/plugins/spermq.md

```diff
@@ -0,0 +1,27 @@
+---
+mediawiki: SpermQ
+title: SpermQ
+description: An ImageJ plugin to analyze the flagellar beat of sperm and sperm steering.
+icon: https://raw.githubusercontent.com/hansenjn/SpermQ/master/Webfiles/20200708%20SpermQ%20Logo%20low%20res.png
+categories: [Automation, Analysis, Segmentation, Cilia, Kymograph]
+doi: 10.3390/cells8010010
+source-url: https://github.com/hansenjn/SpermQ
+dev-status: Stable
+support-status: Active
+team-founder: '@hansenjn'
+team-maintainer: '@hansenjn'
+---
+
+An ImageJ plugin to analyze the flagellar beat of sperm and sperm steering.
+ 
+The software is published with the following publication:
+Hansen, J.N.; Rassmann, S.; Jikeli, J.F.; Wachten, D. 
+*SpermQ–A Simple Analysis Software to Comprehensively Study Flagellar Beating and Sperm Steering.*
+Cells 2019, 8, 10. doi: https://doi.org/10.3390/cells8010010.
```

Comment:
Please use the `{% include citatation}` mechanism to cite publications. See https://imagej.net/editing/citations